torch/nn/_functions/thnn/auto_double_backwards.py

```diff
@@ -106,6 +106,27 @@ def logsoftmax_double_backwards(ctx, ggI):
     return gI, ggO, None, None, None, None
 
 
+def reflectionpad2d_double_backwards(ctx, ggI):
+    gI = None
+    ggO = torch.nn._functions.thnn.auto.ReflectionPad2d.apply(ggI, *ctx.additional_args)
```

Comment:
It might be possible to shorten the chain (say start with `_functions` or `thnn`)

Reply:
played around with it, doesn't seem possible (at least in python 2).